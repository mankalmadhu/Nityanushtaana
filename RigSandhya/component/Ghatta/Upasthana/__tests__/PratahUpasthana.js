import React from 'react';
import ಪ್ರಾತರುಪಸ್ಥಾನ from '../PratahUpasthana';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../JaataVedase',()=>'ಜಾತವೇದಸೇ');
jest.mock('../Tachayo',()=>'ತಚ್ಛ೦ಯೋ');
jest.mock('../NamoBramhane',()=>'ನಮೋಬ್ರಹ್ಮಣೇ');
jest.mock('../PratahDishaNamskara',()=>'ಪ್ರಾತರ್ದಿಶಾನಮಸ್ಕಾರ');
jest.mock('../DevataNamskara',()=>'ದೇವತಾನಮಸ್ಕಾರ');
jest.mock('../Niragamana',()=>'ನಿರ್ಗಮನ');
jest.mock('../Pravara',()=>'ಪ್ರವರ');

test("ಪ್ರಾತರುಪಸ್ಥಾನ", () => {
   expect(Renderer.create(
    <ಪ್ರಾತರುಪಸ್ಥಾನ />
  ).toJSON()).toMatchSnapshot();

});