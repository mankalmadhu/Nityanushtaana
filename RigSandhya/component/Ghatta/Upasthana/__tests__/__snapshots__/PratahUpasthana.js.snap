// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಪ್ರಾತರುಪಸ್ಥಾನ 1`] = `
<View>
  <ಶೀರ್ಷಿಕೆ>
    ಉಪಸ್ಥಾನ
  </ಶೀರ್ಷಿಕೆ>
  <ಶೀರ್ಷಿಕೆ>
    (ಪೂರ್ವಾಭಿಮುಖವಾಗಿ ನಿ೦ತು)
  </ಶೀರ್ಷಿಕೆ>
  <ಜಾತವೇದಸೇ />
  <ತಚ್ಛ೦ಯೋ />
  <ನಮೋಬ್ರಹ್ಮಣೇ />
  <ವಿನಿಯೋಗ>
    (ಮೂರು ಬಾರಿ)
  </ವಿನಿಯೋಗ>
  <ಪ್ರಾತರ್ದಿಶಾನಮಸ್ಕಾರ />
  <ದೇವತಾನಮಸ್ಕಾರ />
  <ನಿರ್ಗಮನ />
  <ನಮೋಬ್ರಹ್ಮಣೇ />
  <ಪ್ರವರ />
  <ಆಸತ್ಯಲೋಕಾತ್ />
</View>
`;
